nlp4all/__init__.py

```diff
@@ -11,6 +11,7 @@
 
 
 app = Flask(__name__)
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
```

Comment:
I added this to avoid the deprecation warning from now on.

Reply:
But at some point should we address that? or low priority?

Reply:
@franciscoabenza I think it's fine now, it's more that sqlalchemy requires you to explicitly say if you want it enabled or disabled, we want it disabled because it introduces a [significant performance overhead](https://flask-sqlalchemy.palletsprojects.com/en/3.0.x/track-modifications/)

Reply:
🥇